Secure-by-design-guide.md

```
@@ -0,0 +1,548 @@
+
+    A2["セキュリティを関心事として捉える"] --> B2["すべての経路を横断的に検証"]
+    B2 --> C2["抜け道自体が設計上存在しない"]
+```
+
+#### CIA-T：4つの古典的なセキュリティ関心事
+
+情報セキュリティの世界では伝統的に「CIA」というモデルが使われますが、本書ではさらに「T（追跡可能性）」を加えた「CIA-T」を紹介しています。
+
+| 頭文字 | 意味 | 説明 | 具体例 |
+|---|---|---|---|
+| **C**onfidentiality（機密性） | 秘密にすべき情報を秘密のまま保つこと | 診療記録が第三者に漏れないこと |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)